src/FixedDataTableRowBuffer.js

```diff
@@ -39,11 +40,13 @@ class FixedDataTableRowBuffer {
     this._viewportRowsBegin = 0;
     this._viewportRowsEnd = 0;
     this._maxVisibleRowCount = Math.ceil(viewportHeight / defaultRowHeight) + 1;
-    this._bufferRowsCount = clamp(
-      Math.floor(this._maxVisibleRowCount/2),
-      MIN_BUFFER_ROWS,
-      MAX_BUFFER_ROWS
-    );
+    this._bufferRowsCount = bufferRowCount !== null
```

Comment:
I think you misunderstood the last thread.
bufferedRows initializes with undefined if the user never passes in this prop right? 
since undefined !== null, this would attempt to use bufferRowCount (undefined) as a buffer